src/Tribe/Views/V2/View.php

```diff
@@ -663,11 +667,59 @@ public function get_html() {
 		return $html;
 	}
 
+	/**
+	 * Returns the view label value after filtering.
+	 *
+	 * This is the method you want to overwrite to replace the label for a view with translations.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public static function get_view_label(): string {
+		return static::filter_view_label( static::$label );
```

Comment:
```suggestion
		return (string)static::filter_view_label( static::$label );
```
Probably less a problem here, but in general I'd do this on these type cast functions that rely on output of filters. People can "accidentally" null it or something that is going to fatal here.

Reply:
Actually this is probably more relevant in the `filter_view_label` function, but you get what I mean.

Reply:
See: https://github.com/the-events-calendar/the-events-calendar/pull/4025/files#diff-b5ac6d676b19cb6b889de2de40d22903ba5bd8d989292eac14d03900e64560e4R715